starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router  = express.Router();
+const  celebModel= require('../models/celebrity');
```

Comment:
as best practices, all models should start with uppercase. So, in this case, something like CelebModel.